docs/pages/getting-started.html

```diff
@@ -33,7 +33,7 @@ <h3 class="usa-heading">For developers</h3>
 
 <ol class="usa-content-list">
   <li>
-    <strong>HTML</strong> markup for the components are located in: 
+    <strong>HTML</strong> markup for the UI Components are located in: 
```

Comment:
Is "markup" meant to be plural here? (I wasn't 100% sure.) If it's not meant to be plural, please change "components are" to "Components is"